ddtrace/contrib/langchain/patch.py

```diff
@@ -603,6 +604,14 @@ def traced_embedding(langchain, pin, func, instance, args, kwargs):
         integration.metric(span, "incr", "request.error", 1)
         raise
     finally:
+        if integration.is_pc_sampled_llmobs(span):
+            integration.llmobs_set_tags(
+                "embedding",
+                span,
+                input_texts=input_texts,
+                output_embedding=func(*args, **kwargs),
```

Comment:
In line 596, we define `embeddings` as the result of the wrapped function. However we should define `embeddings = None` before we try calling the wrapped function, because otherwise it may not exist (in case of error) like we do with `completions` ([ex](https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/contrib/langchain/patch.py#L211))

Reply:
Gotcha !
I tried to get it directly but the issue was that it needed to be initialized before. I was trying to be as less intrusive to the tracing code as possible 😅 But if I can just initialize embeddings before that would be simpler.